Description/Enhancer/Sonata/AdminEnhancer.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\Resource\Description\Enhancer\Sonata;
+
+use Sonata\AdminBundle\Admin\Pool;
+use Doctrine\Common\Util\ClassUtils;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Cmf\Component\Resource\Description\DescriptionEnhancerInterface;
+use Symfony\Cmf\Component\Resource\Description\Description;
+use Puli\Repository\Api\Resource\PuliResource;
+use Symfony\Cmf\Component\Resource\Description\Descriptor;
+
+/**
+ * Add links and meta-info from Sonata Admin.
+ *
+ * @author Daniel Leech <daniel@dantleech.com>
+ */
+class AdminEnhancer implements DescriptionEnhancerInterface
```

Comment:
Still need to "manually" test this or setup a full Sonata environment here for integration testing.